00_Base/src/ocpp/model/schemas/UpdateChargingStationPasswordRequestSchema.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Is this the appropriate location for placing this file?

Reply:
No. This folder is for the automatically generated files that are output by json-schema-processor. We should create a new folder for this class, probably underneath the 'persistence' folder.

Reply:
Got it, I moved it under persistence/schemas, thanks @thanaParis!